bndtools.core/bndtools.gtk.linux-eclipse-neon.x86_64.bndrun

```diff
@@ -0,0 +1,18 @@
+# use this runfile when you are on GTK 3.20+ 
+# because the version from Eclipse 4.4.2 doesnt work with newer versions of GTK
+
+-include: bndtools.shared.bndrun
+
+-runbundles: \
+	${crossplatform.bundles},\
+	org.eclipse.core.filesystem.linux.x86_64,\
+	org.eclipse.core.net.linux.x86_64,\
+	org.eclipse.swt.gtk.linux.x86_64
```

Comment:
it will grab it.  So do this: `version='3.105.2'` this will say "grab at least this version" but higher is fine.